### PR TITLE
Fix broken imagestream trigger example annotation

### PR DESCRIPTION
**Pull request by bparees**
the section/module i'm deleting didn't seem to add anything useful.

but the main change here is to fix the example so that it's actually correct and works.

### CODE REVIEW

**Comment by bparees** [comment]
@bmcelvee ptal. needs to go back as far as it exists (the example was always wrong)

File: modules/images-triggering-updates-imagestream-changes-kubernetes-cli.adoc [hunk 1]
```diff
@@ -4,11 +4,9 @@
 
 
 [id="images-triggering-updates-imagestream-changes-kubernetes-cli_{context}"]
-= Triggering Kubernetes resources manually
+= Setting the Image Trigger on Kubernetes resources
 
-You can manually trigger Kubernetes resources by entering the `oc set triggers` command.
-
-When adding an image trigger to deployments, you can use the `oc set triggers` command. For example, the example command in this procedure adds an image change trigger to the deployment named `example` so that when the `example:latest` image stream tag is updated, the `web` container inside the deployment updates with the new image value.
+When adding an image trigger to deployments, you can use the `oc set triggers` command. For example, the sample command in this procedure adds an image change trigger to the deployment named `example` so that when the `example:latest` image stream tag is updated, the `web` container inside the deployment updates with the new image value.  This command will set the correct `image.openshift.io/triggers` annotation on the deployment resource.
```
Comment (bmcelvee) [inline modules/images-triggering-updates-imagestream-changes-kubernetes-cli.adoc:9]: Double space before "This command"

s/This command will set.../This command sets...

File: modules/images-triggering-updates-imagestream-changes-kubernetes-cli.adoc [hunk 2]
```diff
@@ -4,11 +4,9 @@
 
 
 [id="images-triggering-updates-imagestream-changes-kubernetes-cli_{context}"]
-= Triggering Kubernetes resources manually
+= Setting the Image Trigger on Kubernetes resources
```
Comment (bmcelvee) [inline modules/images-triggering-updates-imagestream-changes-kubernetes-cli.adoc:7]: s/Image Trigger/image trigger

**Comment by bmcelvee** [comment]
@xiuwang would you mind reviewing this PR too, please? Thanks!

File: modules/images-triggering-updates-imagestream-changes-kubernetes-about.adoc [hunk 3]
```diff
@@ -21,7 +21,7 @@ Value:
      "name": "example:latest", <2>
      "namespace": "myapp", <3>
```
Comment (xiuwang) [inline modules/images-triggering-updates-imagestream-changes-kubernetes-about.adoc:22]: The comma in this line need removed

Comment (bparees) [inline modules/images-triggering-updates-imagestream-changes-kubernetes-about.adoc:22]:
```suggestion
     "namespace": "myapp" <3>
```

Comment (bparees) [reply modules/images-triggering-updates-imagestream-changes-kubernetes-about.adoc]: thanks, updated

**Comment by xiuwang** [comment]
LGTM

**Comment by bmcelvee** [comment]
Thank you! Merging now.

**Pull request merged**